CHANGELOG.md

```diff
@@ -1 +1 @@
-See [CHANGELOG/README.md](./CHANGELOG/README.md).
\ No newline at end of file
+See the [1.10 changelog in the master branch](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/CHANGELOG/CHANGELOG-1.10.md).
```

Comment:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/CHANGELOG/CHANGELOG-1.10.md doesn't exist yet. I will create this in master, but this PR doesn't need to wait for this I don't think.